app/package.json

```diff
@@ -2,33 +2,35 @@
   "name": "headlamp",
   "version": "0.2.1",
   "description": "",
-  "main": "electron/src/main.js",
+  "main": "electron/main.js",
   "homepage": ".",
   "productName": "Headlamp",
   "scripts": {
-    "compile-electron": "babel electron --out-dir electron/src/ --extensions '.ts'",
-    "copy-server-binary": "cd ../ && make backend && cp -r ./backend/server ./app/electron/",
-    "start": "npm run copy-server-binary && cd ../ && make run-backend  & cd ../frontend/ && export BROWSER=none && npm start",
-    "build": "npm run compile-electron && npm run copy-server-binary && cd ../frontend/ && PUBLIC_URL='./' npm run build && cp -r ../app/electron/src/. build/electron",
-    "package": "mkdir -p build && cp -r ../frontend/build/* ./build && electron-builder build -c.extraMetadata.main=build/electron/main.js --publish never",
-    "serve-dev": "npm run compile-electron && export ELECTRON_DEV=1 && export ELECTRON_START_URL=http://localhost:3000 && electron .",
-    "publish": "mkdir -p build && cp -r ../frontend/build/* ./build && electron-builder build -c.extraMetadata.main=build/electron/main.js --publish always"
+    "compile-electron": "babel electron --out-dir electron/ --extensions '.ts'",
+    "start": "cd ../ && make backend && make run-backend  & cd ../frontend/ && export BROWSER=none && npm start",
+    "build": "npm run compile-electron && electron-builder --dir --publish never",
+    "package": "npm run compile-electron && electron-builder build --publish never",
+    "serve-dev": "npm run compile-electron && export ELECTRON_DEV=1 && export ELECTRON_START_URL=http://localhost:3000 && electron ."
   },
   "build": {
     "appId": "com.kinvolk.headlamp",
-    "extraMetadata": {
-      "main": "build/electron/main.js"
-    },
+    "beforeBuild": "./scripts/build-backend.js",
     "afterSign": "mac/scripts/notarize.js",
-    "asar": false,
+    "asar": true,
+    "artifactName": "${productName}-${version}-${os}-${arch}.${ext}",
```

Comment:
Do we really want these?

In the other PR https://github.com/kinvolk/headlamp/pull/214 I set this to just 
```json
    "artifactName": "${name}.${ext}",
```

Which makes eg. "Headlamp.app".

Reply:
I do want it in this format as it matches the name we push when we release.